internal/assemble/assemble.go

```diff
@@ -0,0 +1,75 @@
+package assemble
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/canonical/sqlair/internal/parse"
+	"github.com/canonical/sqlair/internal/typeinfo"
```

Comment:
Not in this PR, but in `typeinfo.go`, we might need to change the docstring from:
`// Reflect will return the Info of a given type,` to:
`// TypeInfo will return the Info of a given type,`